### PR TITLE
@luxumbra/playbooks

### PR DESCRIPTION
**Pull request by luxumbra**
I have added the new playbooks section of the wiki and also added Cooopahs playbook.

I did try to achieve this in a way that would allow guest writers to add playbooks via the CMS but the CMS doesn't seem to be working with adding pages. So unfortunately, we still need to go the markdown route. I think we are hitting the limits of what Docusaurus can do without more intensive dev work, especially with wanting to remove the nav in places. 

Better alternatives for the wiki are being looked at and @peth-yursick was saying it would be good to have a blog & newsletter tied into the same platform, which i think Ghost can do. However I am not sure how customisable Ghost is in terms of design & layout? 

Anyway, this is my PR to get the long overdue Playbooks section added to the wiki. Feedback & reviews welcomed. Lux out. 🙏🏻

### CODE REVIEW

File: docs/playbooks/browse.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+<!-- add playbooks in this section  -->
+export const books = [
+  {
+    id: 1,
+    title: "😎 Make it Without Technical Skills",
+    author: "@cooopahtroopa",
+    imgURL: "https://i.imgur.com/BWIoQ9Q.jpg",
+    bookPath: "playbooks/how-to-make-it-without-technical-skills",
+    decription:
+      "Vestibulum ullamcorper mauris at ligula. Phasellus magna. Aenean ut eros et nisl sagittis vestibulum. Fusce fermentum. Nunc sed turpis.",
```
Comment (alalonde) [inline docs/playbooks/browse.md:33]: Lorem ipsum. not sure if this is displayed anywhere currently

File: src/components/playbooks.js [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+//   },
+//   {
+//     id: 2,
+//     title: "Playbook 2",
+//     imgURL: "https://imgur.com/1Kezv9Q.png",
+//     bookPath: "",
+//     decription:
+//       "Vestibulum ullamcorper mauris at ligula. Phasellus magna. Aenean ut eros et nisl sagittis vestibulum. Fusce fermentum. Nunc sed turpis.",
+//   },
+// ];
```
Comment (alalonde) [inline src/components/playbooks.js:39]: I prefer to see commented-out stuff just deleted.

**Review by alalonde — APPROVED** [review]
Looks great! Nice work.

File: src/components/playbooks.js (hunk 2 above)
Comment (luxumbra) [reply src/components/playbooks.js:39]: Point taken, I'll remove.

File: docs/playbooks/how-to-not-get-hacked.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
+---
+title: 📚 How to not get hacked
```
[the remark this answers is not in the document]
Comment (luxumbra) [reply docs/playbooks/how-to-not-get-hacked.md:2]: Yeah, this is a WIP and references to it should not be live.

File: docs/playbooks/how-to-make-it-without-technical-skills.md [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+### 4) Participate in governance
+Every project has a governance forum. Providing feedback and insights on proposals is 100x more valuable than voting ‘Yes’. This means actively circling back to forums to stay up on the latest posts, and disseminating those discussions in Discord so that others are urged to participate too.
+
+The more you can encourage others to partcipate in governance, the beetter the conversation becomes, and the more likely those same people are to come back time and time again.
+
+Instigate discussions around governance and your circle of friends will grow before your eyes!
+
+A list of projects incentivizing governance today include but are not limited to:
+
+* MakerDAO
```
[the remark this answers is not in the document]
Comment (luxumbra) [reply docs/playbooks/how-to-make-it-without-technical-skills.md:66]: Good point. It was a copy-pasta job from Coopahs original piece he wrote and I copied to the wiki. I'll add any missing links.

File: docs/playbooks/browse.md (hunk 1 above)
Comment (luxumbra) [reply docs/playbooks/browse.md:33]: It is not displayed anywhere currently but it's a good point. I'll remove

File: docs/playbooks/how-to-make-it-without-technical-skills.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (luxumbra) [reply docs/playbooks/how-to-make-it-without-technical-skills.md]: Fixed in 3e692ee

File: docs/playbooks/browse.md (hunk 1 above)
Comment (luxumbra) [reply docs/playbooks/browse.md]: fixed in 3e692ee

File: src/components/playbooks.js (hunk 2 above)
Comment (luxumbra) [reply src/components/playbooks.js]: Fixed in 3e692ee

**Pull request merged**